zetasql/fuzzing/protobuf/zetasql_expression_extractor.cc

```diff
@@ -0,0 +1,144 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//'
+
+#include "zetasql/fuzzing/protobuf/zetasql_expression_extractor.h"
+
+#define EXTRACT_DEFAULT(var) Append(var.default_value().content())
```

Comment:
Don't use a macro. This can be a templated function.
https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros

Also, wrap this in an unnamed namespace to limit scope to this file ("internal linkage").
https://google.github.io/styleguide/cppguide.html#Unnamed_Namespaces_and_Static_Variables